python/Makefile

```
@@ -15,8 +17,9 @@ setup: ## Setup the requirements
 
 .PHONY: develop
 develop: setup ## Install Python binding of delta-rs
-	$(info --- Maturin develop ---)
-	maturin develop
+	$(info --- Maturin build ---)
+	maturin build
+	python -m pip install $(TARGET_WHEEL)[devel,pandas]
```

You could also use pip directly.

```suggestion
    pip install $(TARGET_WHEEL)[devel,pandas]
```